data/json/items/comestibles/med.json

```diff
@@ -333,8 +331,6 @@
         "weight" : 240,
         "volume" : 1,
         "price" : 3000,
-        "charges" : 1,
-        "stack_size" : 1,
```

Comment:
That's the way it should work but I'm not certain it will. Swapping `charges: 1` results in no behavioral changes as your just removing the default value. Swapping `stack_size` could (although I agree it shouldn't)